pkg/agave/assets/install.sh

```diff
@@ -66,7 +66,7 @@ EOF
 }
 
 step::70::install-validator() {
-  $APT install zuma-agave-validator zuma-solana-cli
+  $APT install "zuma-agave-validator=$VALIDATOR_VERSION" "zuma-solana-cli=$VALIDATOR_VERSION"
 }
```

Comment:
I don't know if it's fair that to ask the user to always provide a version, if we don't give them easy insight into what versions are available.  I'd say if they don't provide a version, we should install the latest (i.e. no VALIDATOR_VERSION) passed in.